anaconda.spec.in

```diff
@@ -195,6 +195,39 @@ The anaconda-install-env-deps metapackage lists all installation environment dep
 This makes it possible for packages (such as Initial Setup) to depend on the main Anaconda package without
 pulling in all the install time dependencies as well.
 
+%package install-image-deps
+Summary: Installation image specific dependencies
+Requires: udisks2-iscsi
+Requires: libblockdev-plugins-all >= %{libblockdevver}
+# active directory/freeipa join support
+Requires: realmd
+Requires: isomd5sum >= %{isomd5sumver}
+%ifarch %{ix86} x86_64
+Requires: fcoe-utils >= %{fcoeutilsver}
+%endif
+# likely HFS+ resize support
+%ifarch %{ix86} x86_64
+%if ! 0%{?rhel}
+Requires: hfsplus-tools
+%endif
+%endif
+# kexec support
+Requires: kexec-tools
+Requires: createrepo_c
+# run's on TTY1 in install env
+Requires: tmux
+# install time crash handling
+Requires: gdb
+Requires: rsync
+Requires: device-mapper-multipath
+Requires: kpartx
+Requires: nm-connection-editor
```

Comment:
Looks like the nm-something change should not happen after all. Let's drop that part once there's feedback to apply.

Reply:
It should not. It's already a dependency of gui anaconda package.